docs/quickstart.md

```diff
@@ -53,7 +53,7 @@ metrics_exporters = [OutputExporter(metrics_output_directory),
 dataset = HDF5DataSet(
     root = "./",
     database = database,
-    index = range(400),
+    subset = range(400),
```

Comment:
We should insert keys here, but let's just remove `subset` from the quickstart file for now.